test/e2e/handler/error_messages_formatting_test.go

```diff
@@ -73,14 +73,15 @@ func createPolicyAndWaitForEnactmentCondition(policy string, desiredState func()
 	waitForNodesReady()
 
 	By("Waiting for enactment to be failing")
-	policyconditions.EnactmentConditionsStatusEventually(
+	policyconditions.EnactmentConditionsStatusForPolicyEventually(
```

Comment:
This isn't related to the PR I think.

Reply:
It fixes a flake, I have moved it to a separate commit